go-server-server/go/default.go

```diff
@@ -280,8 +280,13 @@ func ConfigInterfaceVlanPost(w http.ResponseWriter, r *http.Request) {
         return
     }
     if vlan_kv != nil {
-        WriteRequestErrorWithSubCode(w, http.StatusConflict, RESRC_EXISTS,
-              "Object already exists: " + vlan_name, []string{}, "")
+        vlan_if_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VLAN_INTF_TB, vlan_name))
+        if vlan_if_kv != nil {
+                vnet_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VNET_TB, vlan_if_kv["vnet_name"]))
```

Comment:
I thot we have a cache to find guid from vnet name.

Reply:
We don't have that mapping in Cache.
All Caches we have are here: https://github.com/sonic-net/sonic-restapi/blob/043b33cd0f0e533aba45efd81f163fc1bb7f5991/go-server-server/go/persistent.go#L440